analyzers/tests/SonarAnalyzer.UnitTest/Rules/SetPropertiesInsteadOfMethodsTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using CS = SonarAnalyzer.Rules.CSharp;
+using VB = SonarAnalyzer.Rules.VisualBasic;
+
+namespace SonarAnalyzer.UnitTest.Rules;
+
+[TestClass]
+public class SetPropertiesInsteadOfMethodsTest
+{
+    private readonly VerifierBuilder builderCS =  new VerifierBuilder<CS.SetPropertiesInsteadOfMethods>();
+
+    [TestMethod]
+    public void SetPropertiesInsteadOfMethods_CS() =>
+        builderCS.AddPaths("SetPropertiesInsteadOfMethods.cs").Verify();
+
+#if NET
+
+    [TestMethod]
+    public void SetPropertiesInsteadOfMethods_CS_Immutable() =>
+        builderCS
```

Comment:
It's better to have the file path on top (first invocation) IMO. I'm not sure if it's a guideline tho, but I saw only tests with addPaths first, I think it's better from a readability point of view. And the first invocation should be inline
```suggestion
        builderCS.AddPaths("SetPropertiesInsteadOfMethods.Immutable.cs")
```